tests/fixtures/server-fixtures.ts

```
@@ -0,0 +1,36 @@
+import { test as base } from './base-fixtures'
+import { waitForCondition } from '../utils/helpers'
+
+type ServerFixtures = {
+  ensureServersRunning: () => Promise<void>
+}
+
+export const serverTest = base.extend<ServerFixtures>({
+  ensureServersRunning: async ({}, use) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)